components/tag/stories/template.js

```diff
@@ -85,36 +113,23 @@ export const TagsDefaultOptions = ({
 	},
 	content: html`
 		${Template(args, context)}
-		${!args.isInvalid ?
-			Template({
+		${args.isRemovable ? "" : Template({
+			...args,
+			isRemovable: true,
+			}, context)}
```

Comment:
If I haven't said it already, this is solution to showing the removable tags in this template is great! 
![swedishchefkiss](https://github.com/user-attachments/assets/7a02e43d-c2a3-42e0-ab7f-d6774ddc954f)